packages/engine-ui/src/components/CreateVehicle.js

```diff
@@ -17,23 +16,16 @@ const CreateVehicle = ({ onSubmit }) => {
     volume: '',
     weight: '',
     timewindow: { start: null, end: null },
-    startPosition: { lat: 61.8172594, lon: 16.0561472 },
-    endDestination: '',
+    startPosition: { lat: 61.8172594, lon: 16.0561472, name: '' },
+    endPosition: { lat: '', lon: '', name: '' },
```

Comment:
lat lon should not be defaulted string, as it is a number in typings

Reply:
Updated.